.github/workflows/deploy.yml

```
@@ -1,12 +1,16 @@
-name: VitePress Pages
-
 on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)